server/src/database/queries/stats.ts

```diff
@@ -560,45 +560,58 @@ export const getBestSongsNbOffseted = (
 ) =>
   InfosModel.aggregate([
     { $match: basicMatch(user._id, start, end) },
-    {
-      $project: { ...getGroupByDateProjection(user.settings.timezone), id: 1 },
```

Comment:
Also, is there a usage for this line that I removed? I'm not sure where it can be useful here

Reply:
Yeah I don't know either. Must be a line I forgot to remove one day. I think it's safe to remove it